helpers.go

```diff
@@ -217,11 +228,23 @@ func searchWordsByTagIds(db *sql.DB, tags string) {
 	populateTagIds(db)
 	removeEmptyTags()
 
-	if len(Tags) > 0 {
-		log.Infoln("Using tags:", Tags)
-		queryStr = fmt.Sprintf(queryStr+" left join wt as t2 on t2.word_id=t1.id WHERE t2.tag_id IN (%s) group by t1.id", TagsToString())
+	if len(Tags) == 0 {
+		log.Fatalln("db is empty, insert some words with tags")
+	}
```

Comment:
this breaks orunmila 
```suggestion
	if len(Tags) == 0 {
		log.Fatalln("db is empty, insert some words with tags")
	}
```